includes/components/class-admin.php

```diff
@@ -1481,6 +1484,40 @@ public function enqueue_scripts() {
 		if ( in_array( $pagenow, [ 'edit-tags.php', 'term.php' ], true ) ) {
 			wp_enqueue_media();
 		}
+
+		// Get pointers.
+		$pointers = apply_filters( 'hivepress/v1/admin/tooltips', [] );
```

Comment:
please use hivepress/v1/components/admin/tooltips instead (we'll change the notices helper the same way later in v2)